style/pages/projects-details.css

```diff
@@ -15,13 +18,78 @@
 }
 
 .projects-details .related-projects-list .searchbar,
-.projects-details .related-projects-list .nav > * {
+.projects-details .related-projects-list .projects > * {
```

Comment:
1. How come you have list inside a list?
2. Who is immediate child of `.projects`? Don't abuse these dynamic selectors. If you know who you're targeting, use their name.